docs/src/pages/components/dividers/VerticalDividers.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import FormatAlignLeftIcon from '@material-ui/icons/FormatAlignLeft';
+import FormatAlignCenterIcon from '@material-ui/icons/FormatAlignCenter';
+import FormatAlignRightIcon from '@material-ui/icons/FormatAlignRight';
+import FormatBoldIcon from '@material-ui/icons/FormatBold';
+import FormatItalicIcon from '@material-ui/icons/FormatItalic';
+import FormatUnderlinedIcon from '@material-ui/icons/FormatUnderlined';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+
+const useStyles = makeStyles(theme => ({
+  toggleContainer: {
+    margin: theme.spacing(2, 0),
+    width: 'fit-content',
+    border: '1px solid rgba(0, 0, 0, 0.12)',
+    borderRadius: 4,
+    backgroundColor: theme.palette.background.paper,
+    '& svg': {
+      color: 'rgba(0, 0, 0, 0.54)',
+      margin: theme.spacing(2),
+    },
+  },
+  divider: {
+    height: 32,
+    width: 1,
+    margin: 4,
+  },
+}));
+
+export default function ToggleButtons() {
+  const classes = useStyles();
+
+  return (
+    <Grid container alignItems="center" className={classes.toggleContainer}>
+      <FormatAlignLeftIcon />
+      <FormatAlignCenterIcon />
+      <FormatAlignRightIcon />
+      <Divider orientation="horizontal" className={classes.divider} />
```

Comment:
I presume orientation should be vertical – and yet this setting doesn't appear to do anything...

Reply:
You're right! When I saw the size style changes in the final version of the ToggleSwitch demo, I played around some to see how the Divider component actually handled styling it and mistakenly left it with height/width set to static values so it would always appear vertical. 
Fixed 👌 

It might be worth considering changing the `vertical` styles in the Divider component itself - it currently sets height to 100%, but this doesn't typically seem to work out of the box without changing it to `auto` and adding `alignSelf: "stretch"` like in the other example.

Reply:
Alternatively, we can let the divider be full height.

Reply:
Like add those additional styles to the actual Divider component or something else? Cause that's what I was thinking, I can do that if you'd like

Reply:
Makes sense to me. @oliviertassinari ?

Reply:
I have rebased the pull request on master, I have also aimed for minimizing the number of LOCs.